src/app.css

```
@@ -127,6 +127,15 @@
   * {
     font-size: 15px;
   }
+  @media (prefers-color-scheme: dark) {
+    .select {
+      &, & select {
+        &::picker(select) {
+          border: var(--border) solid var(--color-base-content);
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: picker(select)

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)